cmd/gcs/main.go

```diff
@@ -84,6 +84,7 @@ func readMemoryEvents(startTime time.Time, efdFile *os.File, cgName string, thre
 func main() {
 	startTime := time.Now()
 	logLevel := flag.String("loglevel", "debug", "Logging Level: debug, info, warning, error, fatal, panic.")
+	coreDumpLoc := flag.String("core-dump-location", "", "The location/format where process core dumps will be written to.")
```

Comment:
why do we allow a format?

Reply:
To allow flexibility for a client. Coredumps have some identifiers you can provide to customize the name. See "Naming of core dump files" https://man7.org/linux/man-pages/man5/core.5.html. We're not allowing something we parse and do anything with, it's just handled by the kernel.